### ISSUE TITLE
another position error with php5

### ISSUE THREAD

**Issue by imuli** [z7zmey/php-parser#38]
Again, the php7 parser produces sane output. This one looks like it's stemming from expr.Variable.

Sorry not to be submiting patches with these, I've never touched yacc before and am finding the parser a bit hard to follow. If I find more of these should I continue opening new bugs or just reopen this one with more info?

```php
<?php
$here->where();
```

```
==> method_call.php
  | [*node.Root]
  |   "Position": Pos{Line: 2-2 Pos: 19-21};    # ought to be 7-21
  |   "Stmts":
  |     [*stmt.Expression]
  |       "Position": Pos{Line: 2-2 Pos: 19-21};    # ought to be 7-21
  |       "Expr":
  |         [*expr.MethodCall]
  |           "Position": Pos{Line: 2-2 Pos: 19-20};    # ought to be 7-20
  |           "Variable":
  |             [*expr.Variable]
  |               "Position": Pos{Line: 2-2 Pos: 7-20};    # ought to be 7-11
  |               "VarName":
  |                 [*node.Identifier]
  |                   "Position": Pos{Line: 2-2 Pos: 7-11};
  |                   "Value": here;
  |           "Method":
  |             [*node.Identifier]
  |               "Position": Pos{Line: 2-2 Pos: 14-18};
  |               "Value": where;
  |           "ArgumentList":
  |             [*node.ArgumentList]
  |               "Position": Pos{Line: 2-2 Pos: 19-20};
```

**Comment by z7zmey** [comment]
Now dump looks like:

```
==> /Users/Vadim/work/go/src/github.com/z7zmey/php-parser/example.php
  | [*node.Root]
  |   "Position": Pos{Line: 2-2 Pos: 7-21};
  |   "Stmts":
  |     [*stmt.Expression]
  |       "Position": Pos{Line: 2-2 Pos: 7-21}; // represents `$here->where();`
  |       "Expr":
  |         [*expr.MethodCall]
  |           "Position": Pos{Line: 2-2 Pos: 7-20}; // represents `$here->where()`
  |           "Variable":
  |             [*expr.Variable]
  |               "Position": Pos{Line: 2-2 Pos: 7-11}; // represents `$here`
  |               "VarName":
  |                 [*node.Identifier]
  |                   "Position": Pos{Line: 2-2 Pos: 7-11}; // represents `$here`
  |                   "Value": here;
  |           "Method":
  |             [*node.Identifier]
  |               "Position": Pos{Line: 2-2 Pos: 14-18}; // represents `where`
  |               "Value": where;
  |           "ArgumentList":
  |             [*node.ArgumentList]
  |               "Position": Pos{Line: 2-2 Pos: 19-20}; // represents `()`
```

Feel free to opening issues. I will fix them as soon as possible. 
Unfortunately, positions do not cover by tests yet.
